mp/src/game/server/props.cpp

```diff
@@ -40,7 +40,13 @@
 #include "doors.h"
 #include "physics_collisionevent.h"
 #include "gamestats.h"
+#define TF_DLL //FIXME
```

Comment:
There is no macro specific to this project. I assume that other code has been changed with the assumption, that no other variant(/game) is compiled from this fork.
To cleanly separate mod-specific changes it may be a good idea to add such a macro though. `TF_DLL` is already taken, `MOD_INVASION` perhaps?
If you don't think this approach I can just `#if 0 [old code] #else [new code] #endif`, or flat out remove the irrelevant code.

Reply:
Personally I've not been concerned about seperating the mod-specific changes as I'm not expecting Valve to ever update the SDK at this stage - I'd prefer just to remove the irrelevant code.

Reply:
It may be interesting to merge other forks, like [Mapbase](https://developer.valvesoftware.com/wiki/Mapbase) (if it gets an MP port, mapbase-source#77).